spec/integration/user_index_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+
+RSpec.describe 'User index page', type: :feature do
+  before(:each) do
+    @user1 = User.create(name: 'Usom Factory', photo: 'https://source.unsplash.com/user/c_v_r/100x100', bio: 'Usom bio')
+    @user2 = User.create(name: 'Burger', photo: 'https://i.picsum.photos/id/216/200/300.jpg?hmac', bio: 'Burger bio')
+
+    visit users_path
+  end
+
+  it 'displays the username of all other users' do
+    expect(page).to have_content('Kotonko')
+    expect(page).to have_content('Usom')
+  end
+
+  it 'should have profile picture content of all other users' do
+    expect(page).to have_css("img[src*='/images/default.jpg']")
+    expect(page).to have_css("img[src*='/images/default.jpg']")
```

Comment:
- [ ] Dear Team, here we are trying to test if we can see the profile picture of `each` user. This means we need to test the profile picture with a different `src` value. 
- [ ] Besides that, I can see you are trying to find a picture with `src*='/images/default.jpg` while in fact, the value of a `photo` attribute for a user is ` photo: 'https://i.picsum.photos/id/216/200/300.jpg?hmac'`. We need to test whether the value is the same as what the user has in the database.